explorer/src/logging.ts

```diff
@@ -1,46 +1,65 @@
-import {
-  requestWhitelist,
-  logger as loggerConfig,
-  errorLogger,
-} from 'express-winston'
-import * as winston from 'winston'
+import PinoHttp from 'express-pino-logger'
+import pino from 'pino'
 import express from 'express'
+import { Logger } from 'typeorm'
 
-const LOGGER_WHITELIST = [
-  'url',
-  'method',
-  'httpVersion',
-  'originalUrl',
-  'query',
-]
-requestWhitelist.splice(0, requestWhitelist.length, ...LOGGER_WHITELIST)
+const options: any = {
+  name: 'Explorer',
+  level: 'warn',
+  redact: {
+    paths: ['req.headers', 'res.headers'],
+  },
+}
+if (process.env.EXPLORER_DEV) {
+  options['prettyPrint'] = { colorize: true }
```

Comment:
Typically we use dot notation for object access, unless its for keys that have invalid syntax like hyphens.

```suggestion
  options.prettyPrint = { colorize: true }
```